components/ILIAS/ResourceStorage/classes/Collections/View/RequestToDataTable.php

```diff
@@ -100,8 +100,8 @@ protected function buildTable(): \ILIAS\UI\Component\Table\Data
             $this->http->request()
         )->withActions(
             $this->action_builder->getActions()
-        )->withNumberOfRows(
-            $this->request->getItemsPerPage()
+        )->withRange(
+            new Range(0, $this->request->getItemsPerPage())
```

Comment:
@nhaagen Wouldn't that mean that only the first X entries are displayed? Or would the pagination itself then update that? I'm just a little confused about the static “0”.

Reply:
@nhaagen Ok, I think I understand it now, it's really just about the default values :-) That's fine with me, thank you very much!